src/version/mod.rs

```diff
@@ -0,0 +1,157 @@
+use std::fs;
+use std::path::PathBuf;
+use std::str::FromStr;
+use std::sync::mpsc;
+use std::thread;
+use std::time::SystemTime;
+
+use crate::settings::global_user::get_wrangler_home_dir;
```

Comment:
nit: this probably does not  belong  to global_user,  but idk where to  put it just now.